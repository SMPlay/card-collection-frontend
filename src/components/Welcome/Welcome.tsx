import React from "react";
import {
  makeStyles,
  Theme,
  createStyles,
} from "@material-ui/core/styles";
import { Typography } from "@material-ui/core";

const useStyles = makeStyles((theme: Theme) => 
  createStyles({
    mainMarginTop: {
      marginTop: theme.spacing(3)
    },
    welcome: {
      marginTop: theme.spacing(5)
    }
  })
);

export const Welcome: React.FC = () => {
  const styles = useStyles();

  return (
    <>
      <Typography variant="h2" component="h1">
        Супергеройские коллекции
      </Typography>
      <article className={styles.welcome}>
        <Typography variant="h3" component="h3">
          Добро пожаловать на наш сайт с коллекциями карт!
        </Typography>
        <Typography className={styles.mainMarginTop} variant="body1" component="p">
          Здесь вы можете просматривать все ваши любимые карточки из детства
          из различных коллекций.
          Также на сайте реализован аукцион, на котором вы можете приобрести
          карточки у других пользователей.
        </Typography>
        <Typography className={styles.mainMarginTop} variant="body1" component="p">
          Для того, чтобы воспользоваться аукционом, достаточно зарегистрироваться
          или войти и перейти во вкладку "Аукцион" в меню.
        </Typography>
      </article>
    </>
  );
};
