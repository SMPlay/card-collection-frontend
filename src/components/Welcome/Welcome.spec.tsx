import React from "react";
import renderer from 'react-test-renderer';

import { Welcome } from './Welcome';

describe("Welcome component", () => {
  it("should render", () => {
    const tree = renderer.create(
      <Welcome/>
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
