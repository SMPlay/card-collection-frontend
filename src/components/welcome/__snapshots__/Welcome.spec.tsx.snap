// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`welcome component should render 1`] = `
Array [
  <h1
    className="MuiTypography-root MuiTypography-h2"
  >
    Супергеройские коллекции
  </h1>,
  <article
    className="makeStyles-welcome-2"
  >
    <h3
      className="MuiTypography-root MuiTypography-h3"
    >
      Добро пожаловать на наш сайт с коллекциями карт!
    </h3>
    <p
      className="MuiTypography-root makeStyles-mainMarginTop-1 MuiTypography-body1"
    >
      Здесь вы можете просматривать все ваши любимые карточки из детства из различных коллекций. Также на сайте реализован аукцион, на котором вы можете приобрести карточки у других пользователей.
    </p>
    <p
      className="MuiTypography-root makeStyles-mainMarginTop-1 MuiTypography-body1"
    >
      Для того, чтобы воспользоваться аукционом, достаточно зарегистрироваться или войти и перейти во вкладку "Аукцион" в меню.
    </p>
  </article>,
]
`;
